.github/workflows/cli-tests.yml

```diff
@@ -30,7 +30,3 @@ jobs:
       - run: npm run test
       - run: npm run lint
       - run: npm run build
-      # Try running each command
-      - run: npx calm validate -p ../calm/pattern/api-gateway.json -i ../calm/samples/api-gateway-implementation.json
-      - run: npx calm generate -p ../calm/pattern/api-gateway.json -s ../calm/draft/2024-04/meta
-      - run: npx calm visualize -p ../calm/pattern/api-gateway.json
```

Comment:
I'm temporarily removing this as I don't think it's working properly. I'm going to readd it once the initial publish is done,